microsoft-365/managed-desktop/service-description/DeviceRegistration.md

```diff
@@ -10,52 +10,44 @@ ms.localizationpriority: medium
 
 Whether you're working with completely new devices or re-using existing ones, to register them with Microsoft Managed Desktop, you'll need to prepare a comma-delimited (CSV) file. This file should include the following information for each device:
 
->Note: This format is for self-service only. Partners have a different format documented [Somewhere else LINK].
+>[!NOTE]
+>You should use this format only if you are registering the devices yourself. If you're working with a Partner, see [Somewhere else LINK].
 
 [//]: # (I see a mention in the Word outline of this CSV file being different depending on the "actor"--can you clarify?)
 [//]: # (Pretty different - the fields for Manufacturer, Model, and Serial have to be EXACT matches, and Hardware Hash is not required at all.)
 
-These values are used for display purposes, and don't need to match properties from the device exactly.
+These values are used for display purposes, and don't need to match properties from the device exactly. For example, it doesn't matter if you list a device model as "Surface" or "Surface Laptop."
+
 - Device manufacturer (example: Microsoft) 
 - Device model (example: Surface Laptop)
 - Device serial number
-
-The Hardware hash must be an exact match.
 - Hardware hash
 
-To obtain the hardware hash you can ask for help from your OEM or Partner, or follow these steps for each device:
-
-1. Open an admin PowerShell window
-2. Install-Script -Name Get-WindowsAutoPilotInfo
-3. Set-ExecutionPolicy -ExecutionPolicy Unrestricted
-4. .\Get-WindowsAutoPilotInfo -OutputFile <path>\hardwarehash.csv
+>[!NOTE]
+>Unlike the other values, the hardware hash must be an exact match.
 
-These steps can also be done on a brand new device before going through OOBE for the first time by following these steps:
+To obtain the hardware hash, ask for help from your OEM or Partner, or follow these steps for each device:
 
-1. On a different device, insert a USB drive
-2. Launch an admin PowerShell window
-3. Save-Script -Name Get-WindowsAutoPilotInfo -Path <pathToUsb>
-4. Turn on the target device, and do not progress OOBE.
-    >Failure to do so will result in needing to reset or reimage the device.
-5. Insert the USB
-6. Hit Shift+F10 
-7. type powershell
-8. cd <pathToUsb>
-9. Set-ExecutionPolicy -ExecutionPolicy Unrestricted
-10. .\Get-WindowsAutoPilotInfo -OutputFile <path>\hardwarehash.csv
-11. shutdown -s -t 0
+1. Start a PowerShell prompt with administrator privileges.
+2. Run `Install-Script -Name Get-WindowsAutoPilotInfo`
+3.Run `Set-ExecutionPolicy -ExecutionPolicy Unrestricted`
+4. Run `.\Get-WindowsAutoPilotInfo -OutputFile <path>\hardwarehash.csv`
 
->Do not power on the target device again until Registration is complete. 
+{and then copy the hardware hash values out of this output .csv into the main one that also has the devices names and stuff, or what?}
```

Comment:
Oh good catch. We're adding support to the website that will just take this format as is. @Magana - you think that will land in the short term? If not, we should totally add a step to merge the files.

Reply:
For now, let's add the step to merge the files, and we can remove the instructions once we're comfortable with other formats

Reply:
roger that. @jaimeo - once we merge I'll add some draft.